docs/reference/aggregations/bucket/adjacency-matrix-aggregation.asciidoc

```diff
@@ -110,5 +110,4 @@ where examining interactions _over time_ becomes important.
 ==== Limitations
 For N filters the matrix of buckets produced can be N²/2 which can be costly.
 The circuit breaker settings prevent results producing too many buckets and to avoid excessive disk seeks
-the `indices.query.bool.max_clause_count` setting is used to limit the number of filters. 
-imposed of 100 filters . This setting can be changed using the `index.max_adjacency_matrix_filters` index-level setting.
```

Comment:
I don't believe this setting exists any more.

Reply:
Apologies for the delay in replying. We used to have a dedicated `max_adjacency_matrix_filters` setting with a default of 100 but changed it to piggy-back off the `indices.query.bool.max_clause_count` setting because they address the same concern - too many query clauses. The second line, "imposed of 100 filters ..." looks like something that should have been deleted when we made that changeover.